src/filter_tree/filter_tree_utils.c

```diff
@@ -10,7 +10,7 @@
 bool isInFilter(const FT_FilterNode *filter) {
 	return (filter->t == FT_N_EXP &&
 			filter->exp.exp->type == AR_EXP_OP &&
-			strcasecmp(filter->exp.exp->op.func_name, "in") == 0);
+			strcasecmp(filter->exp.exp->op.f->name, "in") == 0);
```

Comment:
this is a bit worse, as it breaks opacity, I think we should introduce an inline function which will give you the function name if the expression is of type operation.